var/spack/repos/builtin/packages/icet/package.py

```diff
@@ -17,10 +17,15 @@ class Icet(CMakePackage):
     version('develop', branch='master')
     version('2.1.1', sha256='04cc5b7aa5b3ec95b255febdcfc2312e553ce3db5ca305526803d5737561ec32')
 
+    variant('opengl', default=False, description="Use opengl")
+    variant('shared', default=False, description='Enable shared library')
```

Comment:
```suggestion
    variant('shared', default=True, description='Enable shared library')
```
Usually the default is true? Any reason to change that?

Reply:
I will make 'True' the default. I have put 'False' because before this PR the recipe used static compilation.